crates/hive-utils/scripts/start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Compute the proxy account class hash
+export PROXY_ACCOUNT_CLASS_HASH=$(starkli class-hash ${MADARA_PATH}/cairo-contracts/kakarot/proxy.json)
+
+# Start madara-bin in the background
+/madara-bin \
+    --rpc-external \
+    --rpc-methods=unsafe \
+    --rpc-cors=all \
+    --tmp \
+    --dev &
```

Comment:
These extra flags are needed to run Madara correctly inside a docker container

Reply:
is there a risk associated?
methods unsafe, dev, all these flags seem dangerous haha

Reply:
I think as it's for testing it's fine, it's the same flags in our docker compose 🙂